src/UID2.Client/UID2Client.cs

```diff
@@ -134,7 +134,7 @@ public async Task<RefreshResponse> RefreshAsync(CancellationToken token)
 
         private string GetAssemblyNameAndVersion()
         {
-            var version = "5.4.0";
+            var version = "5.4.7";
```

Comment:
This will need to be incremented some how @jon8787

Reply:
probably change it to 5.4.9 to be in sync with nuspec file at least first

Reply:
right, so the release pipeline increments the version after the pipeline has been kickstarted and before starting to build a new binary and push it to nuget.org, so before we are ready to release, we have to set the version on this line to the next version that we want (the github action pipeline) to release in advance? @cYKatherine ?

Reply:
This conversation has became huge and @jon8787 and I agreed to address this in a separate ticket: https://atlassian.thetradedesk.com/jira/browse/UID2-2964

atm let's keep it manual to update the version here 😢